pkgs/by-name/re/refinery-cli/time-crate.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index 59dad7c..eb5b122 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -460,9 +460,12 @@ dependencies = [
+
+ [[package]]
+ name = "deranged"
+-version = "0.3.8"
++version = "0.3.11"
```

Comment:
Not sure why this updated, but I guess it can't hurt…

Reply:
From what I can tell, the mnimum version is above `0.3.8`, so it was actually required to bump it: https://github.com/time-rs/time/blob/main/Cargo.toml#L11C25-L11C30